specification/crds/go/go.sum

```diff
@@ -0,0 +1,371 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Why is this file needed, sorry i don't understand. is it like a in node a package.json but for go?

Reply:
Exactly! It's the dependency management for go.